src/main/fc/fc_debug.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+typedef enum {
+    DEBUG_NONE,
+    DEBUG_CYCLETIME,
+    DEBUG_NOTCH,
+    DEBUG_GYRO,
+//    DEBUG_BATTERY,
+//    DEBUG_ACCELEROMETER,
+//    DEBUG_MIXER,
+//    DEBUG_AIRMODE,
+    DEBUG_PIDLOOP,
+//    DEBUG_RC_INTERPOLATION,
+//    DEBUG_VELOCITY,
+//    DEBUG_DTERM_FILTER,
+//    DEBUG_ANGLERATE,
+    DEBUG_COUNT
+} debugType_e;
+
+typedef struct debugConfig_s {
```

Comment:
@hydra you dont want to make it harder for loggers by logging everything. I would say that debug is more advanced feature to help you diognose certain parts better. It can be helpfull abalysing peoples problems also when it comes to support.
When users open an issue you can simply ask them to provide more data on this way.

But also the filter tuning can be made easier with more pre filtered data.